### PR TITLE
fix(nav): fix overflowing text from hover. #97

### PR DESCRIPTION
**Pull request by ptbrowne**
https://github.com/cozy/cozy-ui/issues/97

### CODE REVIEW

**Comment by ptbrowne** [comment]
> Same space on the right for every menu item because of the padding of the ul, then add a little padding (like .5em) on items to avoid the text to touch the bottom of this area because it won't be pretty.

Did not understand that.

**Comment by GoOz** [comment]
Yeah sorry, it wasn't really clear. An image is worth 1 000 words they say so…



See, if the word fits perfectly the available space it won't be pretty, there should be an little padding-right on `coz-nav-item` as well just to be safe.

**Comment by ptbrowne** [comment]
OK thanks Understood.

image

File: stylus/ui-app/nav.styl [hunk 1]
```diff
@@ -9,26 +9,36 @@ $navigation
...
 
     .coz-nav-item
-        position  relative
-        z-index   $app-index
+        position        relative
+        z-index         $app-index
+        height          3em
+        display         flex
+        align-items     center
+        padding-right   0.5em
```
Comment (GoOz) [inline stylus/ui-app/nav.styl]: Can you just remove the 0 and write `.5em`? That's what Travis doesn't like. :)

Comment (ptbrowne) [inline stylus/ui-app/nav.styl]: I feel this lint rule is a bit overkill.

**Comment by GoOz** [comment]
Thanks! 👍

**Pull request merged**